### PR TITLE
Schema update & utilization (closes #12 and #13)

### LINKED ISSUE #12: Add reason parameter to q!deny, log who denied the suggestion
This would require modifying the schema.

- [x] Modify schema
- [x] Add parameter to `q!deny`
  - [ ] Add subcommand for setting the reason
- [x] Add proper information to status property

### PR DESCRIPTION
**Pull request by Gorialis**
**This requires a migration!** To perform it, do `psql -d mydb -U myuser < schema.sql` as in the README and the database will be brought up to speed.

This adds timestamps for submission and validation(approval/denial), as well as adding columns related to forced validations, closing #12 and #13.

### CODE REVIEW

**Comment by Gorialis** [comment]
I have already tested this change and it works as intended under all tests I've put it through (force approve, force deny, approve, deny) regarding statuses. I'll commit to add an `update_inplace` to deny but I don't think it actually changes anything in practice.

**Comment by Gorialis** [comment]
After clarification on GitHub, I get what you mean. Fixing hacky properties with a commit now.

**Comment by slice** [comment]
You can rip them out completely if you want.

File: queuebot/cogs/queue/suggestion.py [hunk 1]
```diff
@@ -63,11 +64,19 @@ def status(self):
...
+
+        if self.record["forced_by"]:
+            force_text = f"Forced by: <@{self.record['forced_by']}>\nReason: {self.record['forced_reason']}"
+        else:
+            force_text = ""
+
         return f"""**Suggestion #{self.record['idx']}**
 
-Submitted by <@{self.record['user_id']}>
+Submitted by <@{self.record['user_id']}> at {submission_time}
```
Comment (slice) [inline queuebot/cogs/queue/suggestion.py]: How is the formatting on this? Should we include both 12-hour and 24-hour times?

File: queuebot/cogs/queue/suggestion.py [hunk 2]
```diff
@@ -112,7 +121,7 @@ def status(self):
         """, self.record['idx'])
         await self.update_inplace()
 
-    async def move_to_public_queue(self):
+    async def move_to_public_queue(self, who=None, reason=None):
```
Comment (slice) [inline queuebot/cogs/queue/suggestion.py]: Honestly, I think these should be kwarg-only, at minimum the `reason`, just to match discord.py@rewrite's methods.

File: queuebot/cogs/queue/suggestion.py [hunk 3]
```diff
@@ -166,7 +179,7 @@ def status(self):
             except discord.HTTPException:
                 await self.bot.log(f'\N{WARNING SIGN} Failed to DM `{name_id(user)}` about their approved emoji.')
 
-    async def deny(self):
+    async def deny(self, who=None, reason=None):
```
Comment (slice) [inline queuebot/cogs/queue/suggestion.py]: Ditto.

File: queuebot/cogs/queue/suggestion.py (hunk 1 above)
Comment (Gorialis) [reply queuebot/cogs/queue/suggestion.py]: It's default datetime `__str__`, so this will always be a 24 hour UTC date

**Pull request merged**